playroom/components/index.tsx

```diff
@@ -149,21 +151,30 @@ const PreviewToolsControls = React.forwardRef<HTMLDivElement, PreviewToolsContro
                         value={skinName}
                         onChangeValue={onSkinNameChange as (value: string) => void}
                     />
-                    <Inline space={16} alignItems="center">
+                    <Inline space={0} alignItems="center">
                         {showPlatformSelector && (
-                            <Checkbox
-                                name="iOS"
-                                aria-label="Change platform to iOS"
+                            <ToggleIconButton
+                                checkedProps={{
+                                    Icon: IconAppleOn as React.FC<IconProps>,
+                                    'aria-label': 'Change platform to android',
+                                }}
+                                uncheckedProps={{
+                                    Icon: IconAppleOff as React.FC<IconProps>,
+                                    'aria-label': 'Change platform to iOS',
+                                }}
                                 checked={os === 'ios'}
                                 onChange={(checked) => onOsChange(checked ? 'ios' : 'android')}
-                                render={({checked}) =>
-                                    checked ? <IconAppleOn size={24} /> : <IconAppleOff size={24} />
-                                }
                             />
                         )}
-                        <Checkbox
-                            aria-label="Change color scheme"
-                            name="colorScheme"
+                        <ToggleIconButton
+                            checkedProps={{
+                                Icon: IconSun as React.FC<IconProps>,
+                                'aria-label': 'Change color scheme',
+                            }}
+                            uncheckedProps={{
+                                Icon: IconMoon as React.FC<IconProps>,
+                                'aria-label': 'Change color scheme',
```

Comment:
not important, but maybe a different label for light/dark would be better

Reply:
fixed